Probe fails to connect

```
 docker logs a
Checking for the latest version
Current version 0.2.2
Latest version 0.2.2
[2022-04-02 14:10:51] [INFO] [16] [general] Start probe version 0.2.2 in a production mode
[2022-04-02 14:10:52] [ERROR] [16] [general] connection to API failed failed to collect probe metadata
```

Not sure why it fails since both of our APIs have geo IP data.

```
 curl ipinfo.io
{
  "ip": "139.99.172.23",
  "hostname": "vps-197e15b9.vps.ovh.ca",
  "city": "Sydney",
  "region": "New South Wales",
  "country": "AU",
  "loc": "-33.8678,151.2073",
  "org": "AS16276 OVH SAS",
  "postal": "1001",
  "timezone": "Australia/Sydney",
  "readme": "https://ipinfo.io/missingauth"
```

https://globalping-geoip.global.ssl.fastly.net/139.99.172.23

--- a/src/lib/location/regions.ts
+++ b/src/lib/location/regions.ts
@@ -16,6 +16,7 @@ export const regions = {
 	'central-america': ['BZ', 'CR', 'SV', 'GT', 'HN', 'MX', 'NI', 'PA'],
 	'northern-america': ['CA', 'US', 'BM', 'GL', 'PM'],
 	'southern-america': ['AR', 'BO', 'BR', 'CL', 'CO', 'EC', 'GY', 'PY', 'PE', 'SR', 'UY', 'VE', 'FK', 'GF', 'GS'],
+	australasia: ['AU', 'NZ', 'NF'],
 	caribbean: ['AG', 'BS', 'BB', 'CU', 'DM', 'DO', 'GD', 'HT', 'JM', 'KN', 'LC', 'VC', 'TT', 'GP', 'KY', 'MQ', 'MS', 'TC', 'AW', 'VG', 'VI', 'PR', 'AI', 'MF', 'BL', 'SX', 'CW', 'BQ'],
 	polynesia: ['WS', 'TO', 'TV', 'CK', 'NU', 'PF', 'PN', 'TK', 'WF'],
 	melanesia: ['FJ', 'PG', 'SB', 'VU', 'NC'],

